config/locales/en/activerecord.yml

```diff
@@ -300,6 +300,7 @@ en:
         geozone_restricted: "Restricted by geozone"
         summary: "Summary"
         description: "Description"
+        published: "Publish poll"
```

Comment:
Do you think something like "Published?" would describe better this attribute?

Reply:
Yeah, it makes more sense!

Reply:
Label name was updated.